src/System.IO.Packaging/ref/Configurations.props

```diff
@@ -2,6 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
+      netstandard1.3;
```

Comment:
Just so I understand your logic because in some cases you are adding older versioned netstandard configs and in other cases you are removing them. Are you adding them when you think there is actual implementation we still want to build live for the package and removing them when they are just facades and not interesting to build live?